nshlib/README.md

```diff
@@ -1599,7 +1599,7 @@ rptun     | `CONFIG_RPTUN`
 set       | `CONFIG_NSH_VARS` || !`CONFIG_DISABLE_ENVIRON`
 shutdown  | `CONFIG_BOARDCTL_POWEROFF` || `CONFIG_BOARDCTL_RESET`
 sleep     | -
-source    | `CONFIG_NFILE_STREAMS` > 0 && !`CONFIG_NSH_DISABLESCRIPT`
+source    | `CONFIG_FILE_STREAM` > 0 && !`CONFIG_NSH_DISABLESCRIPT`
```

Comment:
```suggestion
source    | `CONFIG_FILE_STREAM` && !`CONFIG_NSH_DISABLESCRIPT`
```
Here the comparison to 0 does not make sense anymore.

Reply:
Done, thanks.